syntax = "proto3";

package fresnodailynews;

service FresnoDailyNews {
    rpc ExtractKeyword (ExtractKeywordsRequest) returns (ExtractKeywordsResponse) {}
}

message ExtractKeywordsRequest {
    string message = 1;
}

message ExtractKeywordsResponse {
    string reply = 1;
}